NAMESPACE

```diff
@@ -30,6 +30,7 @@ S3method(as.matrix,igraph)
 S3method(as_ids,igraph.es)
 S3method(as_ids,igraph.vs)
 S3method(as_phylo,communities)
+S3method(as_phylo,igraphHRG)
```

Comment:
I wonder why these entries still exist. Should they also be methods for the `ape::as.phylo()` generic and registered via `s3_register()` .

Reply:
I still think this needs to go. All the `as.phylo()` methods should be registered dynamically when/if ape is loaded.

Reply:
But if they don't exist at all how would that not break old  code?